### PR TITLE
Update per-group-guest-access.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a script to toggle back to allow guest access to the group.

I don't think pertains to the subject of the document, however, this is definitely good information for users who read and want to know how to toggle back.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5667

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/solutions/per-group-guest-access.md [hunk 1]
```diff
@@ -62,7 +62,22 @@ Get-AzureADObjectSetting -TargetObjectId $groupID -TargetType Groups | fl Values
 The verification looks like this:
     
 ![Screenshot of PowerShell window showing that guest group access has been set to false.](../media/09ebfb4f-859f-44c3-a29e-63a59fd6ef87.png)
-  
+
+If you wish to toggle the setting back to allow guest access to a particular group, Run the following script, changing */<GroupName/>* to the name of the group where you want to allow guest access.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/per-group-guest-access.md]:
```suggestion
If you wish to toggle the setting back to allow guest access to a particular group, run the following script, changing */<GroupName/>* to the name of the group where you want to allow guest access.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/solutions/per-group-guest-access.md [hunk 2]
```diff
@@ -62,7 +62,22 @@ Get-AzureADObjectSetting -TargetObjectId $groupID -TargetType Groups | fl Values
 The verification looks like this:
     
 ![Screenshot of PowerShell window showing that guest group access has been set to false.](../media/09ebfb4f-859f-44c3-a29e-63a59fd6ef87.png)
-  
+
+If you wish to toggle the setting back to allow guest access to a particular group, run the following script, changing */<GroupName/>* to the name of the group where you want to allow guest access.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/per-group-guest-access.md]:
```suggestion
If you wish to toggle the setting back to allow guest access to a particular group, run the following script, changing ```<GroupName>``` to the name of the group where you want to allow guest access.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies, @msbemba, looking at the rich diff view, I have noticed that this element does not display correctly, so this is my suggestion to correct it. The same correction also needs to be applied to line 42. Many thanks.

**Comment by msbemba** [comment]
@JohanFreelancer9  I shall check line 42 and update if need be.

**Review by JohanFreelancer9 — APPROVED** [review]
@msbemba - Thank you.

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**